src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -880,7 +880,7 @@ static void Main()
                 //             System.Console.Write("catch");
                 Diagnostic(ErrorCode.WRN_UnreachableCode, "System").WithLocation(12, 13)
                 );
-            //CompileAndVerify(comp); // PEVerify failed with Branch out of the method. Follow-up issue: https://github.com/dotnet/roslyn/issues/18678
+            CompileAndVerify(comp);
```

Comment:
The program has unhandled exception, and does not have an output. So I added another test that captures a specific exception outside the function (`PEVerifyWithUnreachableCatch2`). Without the fix, this program will throw `InvalidOperationException` and output `"OtherExceptions"` instead.